mods/d2k/rules/infantry.yaml

```diff
@@ -24,12 +24,40 @@ rifle:
 	AttractsWorms:
 		Intensity: 120
 
+bazooka:
```

Comment:
Could you please move this back to its original location so that the diff shows what has actually changed?